gitlab/resource_gitlab_project_access_token.go

```diff
@@ -0,0 +1,199 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectAccessToken() *schema.Resource {
+	// lintignore: XR002 // TODO: Resolve this tfproviderlint issue
```

Comment:
Please implement the `Importer` instead of disabling [`XR002`](https://github.com/bflad/tfproviderlint/blob/main/xpasses/XR002/README.md)